### ISSUE TITLE
Release Twisted 16.1.1

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#8275]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)| @hawkowl reported|
|-|-|
|Trac ID|trac#8275|
|Type|task|
|Created|2016-04-06 04:34:38Z|
|Branch|https://github.com/twisted/twisted/tree/release-16.1.1-8275|

Patch release containing [#8272](https://github.com/twisted/twisted/issues/8272)

<details><summary>Searchable metadata</summary>

```
trac-id__8275 8275
type__task task
reporter__hawkowl hawkowl
priority__normal normal
milestone__None None
branch__branches_release_16_1_1_8275 branches/release-16.1.1-8275
branch_author__ 
status__closed closed
resolution__fixed fixed
component__release_management release management
keywords__None None
time__1459917278112798 1459917278112798
changetime__1474198049187875 1474198049187875
version__None None
owner__None None
cc__radix
```
</details>

**Comment by twisted-trac** [comment]
|<img alt="uspike's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_3"></a>|uspike commented|
|-|-|

Hi, could this release please include the patch in [#8276](https://github.com/twisted/twisted/issues/8276) ?  It is a relatively small change.  Thanks!

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_4"></a>|@hawkowl commented|
|-|-|

Hi uspike, unfortunately it's not a regression, so I can't. The release cadence should mean that you shouldn't be waiting too long for it to be in a release, though.

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_5"></a>|@hawkowl commented|
|-|-|

Please review.

For reviewers: The test failures are due to an OpenSSH change, and the pyflakes/twistedchecker failures are spurious due to a buildbot change.

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_6"></a>|@adiroiban commented|
|-|-|

I have no idea how to test it.

What is the process for releasing a bugfix release?

I tried

```
git checkout trunk
git merge --squash release-16.1.1-8275
```

but I got a lot of conflicts

The files look good, but in the diff I see a lot of unrelated changes.

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_7"></a>|@hawkowl set status to `closed`|
|-|-|

(In [47206]) Merge release-16.1.1-8275: Release Twisted 16.1.1

Author: hawkowl
Reviewer: adiroiban
Fixes: [#8275](https://github.com/twisted/twisted/issues/8275)

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_8"></a>|@hawkowl commented|
|-|-|

[mass edit] Removing review from closed tickets.
